### PR TITLE
Add multiple chat related functions and offsets

### PR DESCRIPTION
**Pull request by Infiziert90**
All sigs taken from <https://github.com/Infiziert90/ChatTwo/blob/main/ChatTwo/GameFunctions/Chat.cs#L29>

This those not include the print/send functions for tell messages

### CODE REVIEW

**Review by wolfcomp — COMMENTED** [review]
Just some correction stuff that is known if you inspect further down the chain for calls

**Review by wolfcomp — APPROVED** [review]

**Review by wolfcomp — COMMENTED** [review]
The added function also reads and writes to `0xE44` as an `int`

File: FFXIVClientStructs/FFXIV/Client/UI/Misc/AcquaintanceModule.cs [hunk 1]
```diff
@@ -27,6 +27,9 @@ public struct Acquaintance {
         [FieldOffset(0xD8)] public ulong ContentId;
     }
 
+    [MemberFunction("44 8B 89 ?? ?? ?? ?? 4C 8B C1 45 85 C9")]
+    public partial nint GetTellHistory(int index);
```
[the remark this answers is not in the document]
Comment (Haselnussbomber) [reply FFXIVClientStructs/FFXIV/Client/UI/Misc/AcquaintanceModule.cs:31]: FYI, you can suggest changes :joy:
```suggestion
    public partial Aquaitence* GetTellHistory(int index);
```

File: FFXIVClientStructs/FFXIV/Client/UI/Misc/AcquaintanceModule.cs [hunk 2]
```diff
@@ -27,6 +27,9 @@ public struct Acquaintance {
         [FieldOffset(0xD8)] public ulong ContentId;
     }
 
+    [MemberFunction("44 8B 89 ?? ?? ?? ?? 4C 8B C1 45 85 C9")]
+    public partial Aquaitence* GetTellHistory(int index);
```
Comment (Haselnussbomber) [inline FFXIVClientStructs/FFXIV/Client/UI/Misc/AcquaintanceModule.cs:31]: @wolfcomp can't type and I copied it wrong. gg no re
```suggestion
    public partial Acquaintance* GetTellHistory(int index);
```

**Review by Haselnussbomber — CHANGES_REQUESTED** [review]

File: FFXIVClientStructs/FFXIV/Client/UI/Misc/AcquaintanceModule.cs (hunk 2 above)
Comment (wolfcomp) [reply FFXIVClientStructs/FFXIV/Client/UI/Misc/AcquaintanceModule.cs:31]: Whops, but you should know by now to not trust what I type as a copy-paste <.<

**Review by wolfcomp — APPROVED** [review]

**Pull request merged**